en/airframes_experimental/traxxas_stampede.md

```diff
@@ -0,0 +1,122 @@
+# Traxxas Stampede VXL
+
```

Comment:
Great idea! It was chosen mostly because Traxxas RC cars are very popular (see JetsonHacks, MIT racecar just to name two) but I'll add the reasoning here.